lib/railway_ipc/publisher.ex

```diff
@@ -76,12 +78,17 @@ defmodule RailwayIpc.Publisher do
   def prepare_message(message) do
     {:ok, message} =
       message
-      |> Map.put(:uuid, UUID.uuid1())
+      |> ensure_uuid()
       |> Payload.encode()
 
     message
   end
 
+  def ensure_uuid(%{uuid: uuid} = message) when is_nil(uuid) or "" == uuid do
+    Map.put(message, :uuid, UUID.uuid1())
```

Comment:
We can probably rely on ecto uuids generation instead of the uuid lib. Lets remove it.